### PR TITLE
New version: SomePythonThings.WingetUIStore version 1.6.2

### PR DESCRIPTION
**Pull request by marticliment**
###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/microsoft/winget-pkgs/pull/97923)

### CODE REVIEW

**Comment by Otiel** [comment]
Hi @marticliment 
Is v1.6.2-r2 not stable enough to publish it here?

**Comment by marticliment** [comment]
yes it is, is the second version of the installer (the first published version of the installer of the version 1.6.2 had issues that are now fixed.

**Comment by marticliment** [comment]
checks have always failed for this appiication

File: manifests/s/SomePythonThings/WingetUIStore/1.6.2/SomePythonThings.WingetUIStore.installer.yaml [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+Scope: user
+InstallModes:
+- interactive
+- silent
+- silentWithProgress
+UpgradeBehavior: install
+ReleaseDate: 2023-02-27
+Installers:
+- Architecture: x64
+  InstallerUrl: https://github.com/marticliment/WingetUI/releases/download/1.6.2/WingetUI.Installer.exe
```
Comment (Otiel) [inline manifests/s/SomePythonThings/WingetUIStore/1.6.2/SomePythonThings.WingetUIStore.installer.yaml:18]: @marticliment Can you update this PR and reopen it? URL should be `https://github.com/marticliment/WingetUI/releases/download/1.6.2-r2/WingetUI.Installer.exe`

Comment (marticliment) [reply manifests/s/SomePythonThings/WingetUIStore/1.6.2/SomePythonThings.WingetUIStore.installer.yaml]: Done!

**Comment by marticliment** [comment]
Shall I close the other pull request, then? (the one for v1.6.2-r2)

**Comment by Otiel** [comment]
> Shall I close the other pull request, then? (the one for v1.6.2-r2)

I didn't see another PR.

**Comment by marticliment** [comment]
#97925

**Comment by Otiel** [comment]
Ah I see. I searched for [WingetUI|https://github.com/microsoft/winget-pkgs/pulls?q=is%3Apr+wingetui] only and it didn't pop up. Sorry for that, and thanks for your hard work!

**Comment by stephengillie** [comment]
I'm not sure, but I think validation might be getting stuck on initialization questions:
image

**Review by stephengillie — APPROVED** [review]

**Pull request merged**